resources/views/dashboard/tabs/import.php

```diff
@@ -0,0 +1,25 @@
+<div id="import__database" class="text-dark border ml-2">
+    <div class="import__database content p-4">
+        <div class="lead mb-4"><i class="fas text-primary fa-cloud-upload-alt"></i> Import Database:</div>
+        <form enctype="multipart/form-data" method="POST" action="<?= app('url_root') ?>/import" class="d-flex justify-content-between align-items-center">
+            <div class="form-group">
+                <label for="sql_file">Upload file:</label>
+                <input type="file" name="sql_file" id="sql_file">
+                <input type="hidden" name="db_name" value="<?= $_GET['db_name'] ?? '' ?>">
```

Comment:
You can use the Request class and get the values from the GET request rather than using the global `$_GET` object.